src/PlusDataCollection/WinProbe/vtkPlusWinProbeVideoSource.h

```diff
@@ -25,148 +25,165 @@
 class vtkPlusDataCollectionExport vtkPlusWinProbeVideoSource : public vtkPlusUsDevice
 {
 public:
-    /*! Constructor for a smart pointer of this class*/
-    static vtkPlusWinProbeVideoSource * New();
-    vtkTypeMacro(vtkPlusWinProbeVideoSource, vtkPlusDevice);
-    virtual void PrintSelf(ostream& os, vtkIndent indent) VTK_OVERRIDE;
+  /*! Constructor for a smart pointer of this class*/
+  static vtkPlusWinProbeVideoSource* New();
+  vtkTypeMacro(vtkPlusWinProbeVideoSource, vtkPlusDevice);
+  virtual void PrintSelf(ostream& os, vtkIndent indent) VTK_OVERRIDE;
 
-    /*! Specify the device connected to this class */
-    virtual bool IsTracker() const { return false; }
+  /*! Specify the device connected to this class */
+  virtual bool IsTracker() const { return false; }
 
-    /*! Read configuration from xml data */
-    virtual PlusStatus ReadConfiguration(vtkXMLDataElement* config);
+  /*! Read configuration from xml data */
+  virtual PlusStatus ReadConfiguration(vtkXMLDataElement* config);
 
-    /*! Write configuration to xml data */
-    virtual PlusStatus WriteConfiguration(vtkXMLDataElement* config);
+  /*! Write configuration to xml data */
+  virtual PlusStatus WriteConfiguration(vtkXMLDataElement* config);
 
-    /* Set the transmit frequency of US probe (MHz) */
-    PlusStatus SetTxTxFrequency(float frequency);
+  /* Set the transmit frequency of US probe (MHz) */
+  PlusStatus SetTxTxFrequency(float frequency);
 
-    /* Get the transmit frequency of US probe (MHz) */
-    float GetTxTxFrequency();
+  /* Get the transmit frequency of US probe (MHz) */
+  float GetTxTxFrequency();
```

Comment:
Duplicate on purpose?

Reply:
Short answer: yes.

Long answer: WinProbeSDK exposes a lot of internal parameters, and uses prefixes for groups of parameters to avoid name collisions. E.g. `BMode::IsEnabled` gets exposed as `GetBIsEnabled` and `SpatialCompound::IsEnabled` gets exposed as `GetSCIsEnabled`. So `Transmit` class gets prefix `Tx`, and property `TxFrequency` gets exposed as `GetTxTxFrequency`.